svg/shapes/reftests/disabled-shapes-01.svg

```diff
@@ -4,7 +4,7 @@
     <h:link rel="help" href="https://svgwg.org/svg2-draft/shapes.html#RectElement"/>
     <h:link rel="help" href="https://svgwg.org/svg2-draft/shapes.html#CircleElement"/>
     <h:link rel="help" href="https://svgwg.org/svg2-draft/shapes.html#EllipseElement"/>
-    <h:link rel="match" href="support/empty.svg"/>
+    <h:link rel="match" href="about:blank"/>
```

Comment:
I'd rather not rely on `about:blank` as a reference in SVG ref tests, since these need to work for non-browser SVG rendering environments.

(So, keep empty.svg, moved to an appropriately named references folder, which maybe could be `svg/reference/` to create some shared common references.)

Reply:
How much effort would it be to special-case `about:blank` in any reftest runner for other implementations (you could just replace `about:blank` with a URL to an empty SVG)? Practically it should be equivalent to an empty SVG document, but using `about:blank` should improve caching behaviour for browsers given they're more likely to already have a cached rendering of about:blank.

Reply:
I am sceptical about `about:blank` as well. I don't think it is feasible to request implementations or test suites to replace it with something like an empty SVG file.